src/App.jsx

```diff
@@ -0,0 +1,18 @@
+import './styles/App.css'
+import Header from  './header.jsx'
+import Body from './body.jsx'
+import Profile from './profile.jsx'
+
+function App() {
+  return (
+    <>
```

Comment:
messy code; this should be formatted nicely using the built in formatter in VS Code